pages/auction/[auction].tsx

```diff
@@ -1,26 +1,25 @@
-import React, { useEffect, useState } from 'react'
-import AuctionDetails from '../../components/AuctionDetails/AuctionDetails'
-import { useForceUpdate } from '../../utils/Hooks'
+import moment from 'moment'
+import { useRouter } from 'next/router'
+import { useEffect, useState } from 'react'
 import { Container } from 'react-bootstrap'
+import { initAPI } from '../../api/ApiHelper'
+import AuctionDetails from '../../components/AuctionDetails/AuctionDetails'
 import Search from '../../components/Search/Search'
-import { useRouter } from 'next/router'
-import api, { initAPI } from '../../api/ApiHelper'
+import { getCacheControlHeader } from '../../utils/CacheUtils'
+import { numberWithThousandsSeperators } from '../../utils/Formatter'
+import { useForceUpdate } from '../../utils/Hooks'
 import { parseAuctionDetails } from '../../utils/Parser/APIResponseParser'
 import { getHeadElement } from '../../utils/SSRUtils'
-import { numberWithThousandsSeperators } from '../../utils/Formatter'
-import moment from 'moment'
-import { ConstructionOutlined, RttTwoTone } from '@mui/icons-material'
-import { getCacheControlHeader } from '../../utils/CacheUtils'
 
 interface Props {
     auctionDetails: any
 }
 
 function AuctionDetailsPage(props: Props) {
     const router = useRouter()
-    let auctionUUID = router.query.auctionUUID as string
+    let auctionUUID = router.query.auction as string
```

Comment:
Any reason why you changed from "auctionUUID" to "auction"? I think auctionUUID is more descriptive to instantly know what the route contains

Reply:
oh i forgot to change it back, I was doing some debugging in my code, so i tried changing the query name (obviously it didn't)